src/data/AnnoItemFactory.ts

```diff
@@ -91,6 +104,105 @@ export default class AnnoItemFactory {
       .filter((target: EffectTarget) => target.label);
   }
 
+  private resolveTrader(values: any) {
+    let o: Array<any> = [];
+    Object.entries(this.traderItems).forEach((keyval) => {
```

Comment:
you could rewrite this function to:
`return Object.entries(this.traderItems).filter(([key, values]) => values.includes(values.Standard.GUID)).map(([key, values]) => key)`